docs/flow/modin/core/io/index.rst

```diff
@@ -6,34 +6,33 @@ IO Module Description
 Dispatcher Classes Workflow Overview
 ''''''''''''''''''''''''''''''''''''
 
-Call from ``read_*`` function of execution-specific IO class (for example, ``PandasOnRayIO`` for
-Ray engine and pandas storage format) is forwarded to the ``_read`` function of file
+Calls from ``read_*`` functions of execution-specific IO classes (for example, ``PandasOnRayIO`` for
+Ray engine and pandas storage format) are forwarded to the ``_read`` function of the file
 format-specific class (for example ``CSVDispatcher`` for CSV files), where function parameters are
-preprocessed to check if they are supported (otherwise default pandas implementation
-is used) and compute some metadata common for all partitions. Then file is splitted
-into chunks (mechanism of splitting is described below) and using this data, tasks
-are launched on the remote workers. After remote tasks are finished, additional
-results postprocessing is performed, and new query compiler with imported data will
+preprocessed to check if they are supported (defaulting to pandas if not)
+and common metadata is computed for all partitions. The file is then split
+into chunks (splitting mechanism described below) and the data is used to launch tasks
+on the remote workers. After the remote tasks finish, additional
+postprocessing is performed on the results, and a new query compiler with the imported data will
 be returned.
 
 Data File Splitting Mechanism
 '''''''''''''''''''''''''''''
 
-Modin file splitting mechanism differs depending on the data format type:
+Modin's file splitting mechanism differs depending on the data format type:
 
-* text format type - file is splitted into bytes according user specified needs.
+* text format type - the file is split into bytes according to user specified arguments(?).
```

Comment:
What does `?` mean here?

Reply:
It is a typo. See https://github.com/modin-project/modin/pull/4540

Reply:
I see. Thanks!